Security.Platform.Android/Checks/DeveloperModeCheck.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using Android.Content;
+using Android.Provider;
+
+namespace AeroGear.Mobile.Security.Checks
+{
+    public class DeveloperModeCheck : ISecurityCheck
+    {
+        private const string NAME = "Developer Mode Check";
+
+        private readonly Context context;
+
+        public DeveloperModeCheck(Context ctx)
+        {
+            this.context = ctx;
+        }
+
+        public string GetId()
+        {
+            return typeof(DeveloperModeCheck).FullName;
```

Comment:
This is going to be the same for all the checks.
I suggest creating a base class `AbstractSecurityCheck` that implements `ISecurityCheck`
In that class we could add this:
```c#
public string GetId()
{
   return this.GetType().FullName;
}
```